dockerfiles/Dockerfile.sandbox

```diff
@@ -16,11 +16,13 @@
 FROM tiangolo/uwsgi-nginx-flask:python3.10
 
 # Install dependencies required for Lean 4, pypy3, and other tools
+ARG TARGETARCH
```

Comment:
Docker build already provides the specified platform variables as argument. See `TARGET*` variables https://docs.docker.com/build/building/variables/#multi-platform-build-arguments.

Can we instead reuse them?